wp-content/themes/twentytwenty-child/functions.php

```diff
@@ -557,9 +558,79 @@ function search_oxford_dictionary(){
 /** 
  * Creating shortcode for students
  * */
-function students_shortcode() {
+function students_shortcode( $attributes ) {
+    ob_start();
+    $shortcode_args = shortcode_atts( array(
+        'number-of-students' => '',
+        'student-id'         => ''
+    ), $attributes );
+
+    $args = array(
+        'post_type'      => 'student',
+        'p'              => $shortcode_args[ 'student-id' ],
+        'posts_per_page' => $shortcode_args[ 'number-of-students' ]
+    );
+    $query = new WP_Query( $args );
+    
+    if( $query->have_posts() ) {
+        while ( $query->have_posts() ) {
+    ?>
+    <div class="student-box"> 
+        <?php
+            $query->the_post();
+            $data = get_student_info( get_the_ID() );
+
+            echo '<div class="student-name"> ' . get_the_title() . ', ' . $data[ 'class' ] . ' Grade </div>';
+            echo '<div class="student-thumbnail">' . get_the_post_thumbnail() . '</div>';
+        ?>
+    </div>
+    <?php   	
+        }
+    }
+    if( $query->found_posts > $shortcode_args[ 'number-of-students' ] ) {
+        $displayed = $shortcode_args[ 'number-of-students' ];
+        echo load_show_more_button( $displayed, $query->found_posts );
+    }
+    wp_reset_postdata();
+
+    ?>
+    
+    <?php
+
+    return ob_get_clean();
+}
+add_shortcode( 'students', 'students_shortcode' );
+
+function load_show_more_button( $displayed, $found ) {
+    echo '<form method="post"> <div class="show-more-div"> <button class="show-more" value1="' . $displayed . '" value2="' . $found . '" name="show-more">Show more</button> </div> </form>';
```

Comment:
we need to remove the <form> and make it as a typical `<a>` button or a `<button>`